### PR TITLE
[R4ML-{202,224}] 1)sparkR session not deleted 2) setLogLevel msg shou…

### PR DESCRIPTION
**Pull request by aloknsingh**
…ld be at r4ml.session()

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: R4ML/R/zzz.R [hunk 1]
```diff
@@ -355,7 +363,12 @@ r4ml.session.stop <- function() {
   # clean sparkR
   sparkr.stop <- function() {
      if("sc" %in% ls(.GlobalEnv)) { rm(sc, envir = .GlobalEnv) }
-     SparkR::sparkR.session.stop()
+     # only remove if r4ml started the sparkr session
+     if (identical(SparkR:::.sparkREnv, r4ml.env$sparkr.env)) {
```
Comment (bdwyer2) [inline R4ML/R/zzz.R:362]: I don't think this is necessary. There is no harm in calling `SparkR::sparR.session.stop()` regardless of whether a session exists or not.

**Review by bdwyer2 — CHANGES_REQUESTED** [review]
I don't think this PR fixes the issue at hand. I think we need to address `r4ml.env$R4ML_SESSION_EXISTS`.

File: R4ML/R/zzz.R (hunk 1 above)
Comment (aloknsingh) [reply R4ML/R/zzz.R:362]: I think this forces that it will only delete if it owns it. more cleaner

**Comment by aloknsingh** [comment]
@bdwyer2  I agree i miss to remove the R4ML_SESSION check from r4ml.session.stop. I will remove it

File: R4ML/R/zzz.R (hunk 1 above)
Comment (bdwyer2) [reply R4ML/R/zzz.R:362]: I think eventually we should try to find a way to not use `r4ml.session()` and have the user just call `sparkR.session()`

Comment (aloknsingh) [reply R4ML/R/zzz.R:362]: I think, we can't as they are two different tools so it needs to have it's own init.

**Pull request merged**